### PR TITLE
New recipe: OpenLibm, v0.7.5

### PR DESCRIPTION
**Pull request by vazub**

### CODE REVIEW

File: sci-libs/openlibm/openlibm-0.7.5.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+DESCRIPTION="OpenLibm is an effort to have a high quality, portable, \
+standalone C mathematical library (libm). It can be used standalone in \
+applications and programming language implementations.
+
+The project was born out of a need to have a good libm for the Julia \
+programming langage that worked consistently across compilers and operating \
+systems, and in 32-bit and 64-bit environments."
+HOMEPAGE="https://openlibm.org"
+COPYRIGHT="2011-14 The Julia Project. \
+2008 Stephen L. Moshier \
```
Comment (extrowerk) [inline sci-libs/openlibm/openlibm-0.7.5.recipe:11]: should be indented with a tab

**Comment by extrowerk** [comment]
Build / package / test ok on 64 bit.

File: sci-libs/openlibm/openlibm-0.7.5.recipe (hunk 1 above)
Comment (vazub) [reply sci-libs/openlibm/openlibm-0.7.5.recipe:11]: You mean the copyright holders on each separate line?

Comment (extrowerk) [reply sci-libs/openlibm/openlibm-0.7.5.recipe:11]: yep. But only the extra lines.

**Comment by extrowerk** [comment]
Looks good to me.
Care to make a recipe for MKL / aocl-libm-ose?

**Comment by vazub** [comment]
@extrowerk maybe at some point, but not at this time, as right now I am specifically targeting Julia dependencies (openlibm is one of those) to get the port working.

File: sci-libs/openlibm/openlibm-0.7.5.recipe [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+The project was born out of a need to have a good libm for the Julia \
+programming langage that worked consistently across compilers and operating \
+systems, and in 32-bit and 64-bit environments."
+HOMEPAGE="https://openlibm.org"
+COPYRIGHT="2011-14 The Julia Project. \
+	2008 Stephen L. Moshier \
+	1992-2011 The FreeBSD Project. \
+	1993 by Sun Microsystems, Inc. \
+	1997, 1999 Free Software Foundation, Inc."
+LICENSE="OpenLibm"
```
Comment (Begasus) [inline sci-libs/openlibm/openlibm-0.7.5.recipe:15]: MIT, BSD (2-clause), ICS and GNU LGPL v2.1 should be added here too

https://github.com/JuliaMath/openlibm/blob/master/LICENSE.md

**Comment by Begasus** [comment]
Any chance on adding 32bit support here?

File: sci-libs/openlibm/openlibm-0.7.5.recipe [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	openlibm_devel = $portVersion
+	devel:libopenlibm = $libVersionCompat
+	"
+REQUIRES_devel="
+	haiku_devel
```
Comment (Begasus) [inline sci-libs/openlibm/openlibm-0.7.5.recipe:39]: This should be:
```diff
- 	haiku_devel
+ 	openlibm == $portVersion base
```

File: sci-libs/openlibm/openlibm-0.7.5.recipe (hunk 2 above)
Comment (Begasus) [reply sci-libs/openlibm/openlibm-0.7.5.recipe:15]: Seeing they are mentioned in the OpenLibm maybe not, @korli any thoughts (I see other ports at repology mentioning them)

Comment (vazub) [reply sci-libs/openlibm/openlibm-0.7.5.recipe:15]: This is a custom mix of licenses, therefore "OpenLibm" as a placeholder. The document outlines various caveats there, which will be lost if just replacing it with other generic license templates.

Comment (korli) [reply sci-libs/openlibm/openlibm-0.7.5.recipe:15]: Combining different licenses doesn't make any difference. Openlibm uses multiple licenses for different parts, they should be detailed.

Comment (vazub) [reply sci-libs/openlibm/openlibm-0.7.5.recipe]: Fixed

File: sci-libs/openlibm/openlibm-0.7.5.recipe (hunk 3 above)
Comment (vazub) [reply sci-libs/openlibm/openlibm-0.7.5.recipe]: Fixed

**Comment by vazub** [comment]
> 
> 
> Any chance on adding 32bit support here?
> **EDIT** checked a local build and it builds/packages, only the pkg-config file should be looked in, points to the wrong directories

Not at this time, sorry. Maybe with the future iteration.

**Review by korli — APPROVED** [review]

**Comment by Begasus** [comment]
Added 32bit support in https://github.com/haikuports/haikuports/pull/6112

**Pull request merged**